### ISSUE TITLE
Hero Section Image Changes

### ISSUE THREAD

**Issue by eessex** [artsy/positron#1296]
The hero section previously supported only one image, but is unclear whether this is a style decision or because the image_collection component did not yet exist when it was built. 

For article2 the hero section is only accessible to internal team channels, and because of its limited access I'm going to go ahead and use the image_collection component, meaning multiple images can be included in the hero, and deprecate the image component entirely. 

To support this we will need to:
- [ ] Backfill Hero Sections on non-editorial channels
- [ ] Confirm that hero images are rendering correctly in Force
- [ ] Double check how hero images are rendered in AMP 
       *maybe tangential, as team channels don't seem to be distributed to AMP

cc @owendodd

**Comment by owendodd** [comment]
yeah I think it is ok for header to support multiple images. Feels like we wouldn't really want artworks in the header though... Is that a tricky thing to restrict?

**Comment by eessex** [comment]
Should be straightforward to remove them, thanks!
